ui/webui/src/app.jsx

```diff
@@ -46,6 +53,46 @@ export const Application = () => {
         return null;
     }
 
+    const wrapWithContext = children => {
+        return (
+            <AddressContext.Provider value={address}>
+                {children}
+            </AddressContext.Provider>
+        );
+    };
+
+    const steps = [
+        {
+            name: _("Installation language"),
+            component: wrapWithContext(<InstallationLanguage />),
+            stepNavItemProps: { id: "installation-language" }
+        },
+        {
+            name: _("Storage configuration"),
+            component: wrapWithContext(<InstallationDestination />),
+            stepNavItemProps: { id: "installation-destination" }
+        },
+        {
+            name: _("Review"),
+            component: wrapWithContext(<ReviewConfiguration />),
+            nextButtonText: _("Begin installation"),
```

Comment:
Can you check if there is a way to disable the 'Next' button while installation is running. (Can be done also on followup)

Reply:
Lets do that in a followup, so we have this in quickly and followup work can happen more easily in parallel. :)

Reply:
This is implemented now.